effect/index.d.ts

```diff
@@ -0,0 +1,40 @@
+import type { StoreValues } from '../computed/index.d.ts'
+import type { AnyStore, Store, StoreValue } from '../index.js'
+
+interface Effect {
+  <OriginStore extends Store>(
+    stores: OriginStore,
+    cb: (value: StoreValue<OriginStore>) => void | VoidFunction
+  ): VoidFunction
+  /**
+   *
+   * Create effect, which subscribes to all stores and calls callback with their values on change in any of them.
+   * Especially useful for side effects which must cleanup after themselves.
+   *
+   * Example of resize observer effect, which depends on the element and observer options atoms
+   * ```js
+   * let $height = atom(0)
```

Comment:
It will be impossible to see this example in IDE popup

Reply:
Why is it? I see it normarly. Or do you mean it is too big as an example in JSDoc?

Reply:
Simplified examples, both in README and JSDoc. Could you check it?